dhis-2/dhis-api/src/main/java/org/hisp/dhis/relationship/RelationshipType.java

```diff
@@ -58,6 +59,10 @@
 
     private String toFromName;
 
+    private transient String displayFormToName;
```

Comment:
Typo in the name: displayFromToName, not displayFormToName

Reply:
fixed